pkgs/top-level/all-packages.nix

```diff
@@ -17548,8 +17548,9 @@ in
     python = python37;
   };
 
-  protobuf = protobuf3_16;
+  protobuf = protobuf3_17;
 
+  protobuf3_17 = callPackage ../development/libraries/protobuf/3.17.nix { };
   protobuf3_16 = callPackage ../development/libraries/protobuf/3.16.nix { };
   protobuf3_15 = callPackage ../development/libraries/protobuf/3.15.nix { };
   protobuf3_14 = callPackage ../development/libraries/protobuf/3.14.nix { };
```

Comment:
We can probably drop 3.14 now.

Reply:
yes, I think we can drop most of these versions. I'm going to address it in another PR